### ISSUE TITLE
GammaRay exits after 1 minute

### ISSUE THREAD

**Issue by kpopv** [KDAB/GammaRay#960]
I built GammaRay 3.0 for Qt 5.15 (64-bit) with MinGW 8.10 (comes with Qt 5.15) on Windows 11. The selftest indicates everything is ok.

When launching an application, built with the same kit, the launcher starts the gammaray.exe process that in turn starts the application correctly, but no GammaRay UI is shown. The application works as expected, but after about 1 minute, the gammaray.exe process exit and closes the application without any indication what went wrong. There is no logging (also checked with DebugView).

Any suggestions on how to diagnose what is going wrong?

**Comment by Waqar144** [comment]
Yeah, I would suggest to try to use the debugger to see what's happening.

**Comment by kpopv** [comment]
This indicates "Target not responding - timeout". Injection fails.
